static/src/javascripts/projects/common/modules/experiments/tests/cmp-ccpa-test.js

```diff
@@ -0,0 +1,27 @@
+// @flow strict
+
+export const ccpaCMPTest: ABTest = {
+    id: 'CmpCcpaOptIn',
+    start: '2020-06-08',
+    expiry: '2020-06-30',
```

Comment:
should expiryDate be same with sellByDate?

Reply:
Good catch. I'll update this